spec/factories.rb

```diff
@@ -10,8 +10,9 @@
   end
 
   factory :appointment do
-    starts_at DateTime.current + 1.week
-    ends_at DateTime.current + 1.week + 2.hours
+    date_on Date.tomorrow
+    starts_at Time.now.strftime('%H:%M')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.